beard/metrics/string_distance.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
```

Comment:
I used `string_methods.py` for a temporary name until we find something more meaningful.